lib/msal-browser/src/index.ts

```diff
@@ -12,5 +12,7 @@ export {
     // Error
     AuthError,
     AuthErrorMessage,
-    INetworkModule
+    INetworkModule,
+    // Logger Object
+    Logger
```

Comment:
I think `LogLevel` should also be exported, in case someone wants to write logic based on that for each message.

Reply:
Done